source/renderer/app/api/utils/patchAdaApi.js

```diff
@@ -50,6 +55,10 @@ export default (api: AdaApi) => {
           epochNumber: get(next_epoch, 'epoch_number', 0),
           epochStart: get(next_epoch, 'epoch_start', ''),
         },
+        futureEpoch: {
```

Comment:
@daniloprates please refactor to match: https://github.com/input-output-hk/daedalus/pull/1762/files#diff-6aa9e91ae277a5d21b8e1bb0b03ffafbR1167

Reply:
✅